From fe0780145bf0f9fba79d9d21fd2b76c4d4edc4bc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Fri, 4 Mar 2016 16:45:29 +0000
Subject: [PATCH] Resolves: tdf#98399 RET_USER_CANCEL is non-zero and returned
 on cancel

Change-Id: Id4231454dfb34cac6ee89d63beb18a83c63a44c1
(cherry picked from commit cd9a5cf4312a2dc0c1ecbf682c67ca08862cdde0)
---
 sw/source/uibase/shells/grfsh.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sw/source/uibase/shells/grfsh.cxx b/sw/source/uibase/shells/grfsh.cxx
index 39e18fd..abaa41e 100644
--- a/sw/source/uibase/shells/grfsh.cxx
+++ b/sw/source/uibase/shells/grfsh.cxx
@@ -354,7 +354,7 @@ void SwGrfShell::Execute(SfxRequest &rReq)
             if (nSlot == FN_DRAW_WRAP_DLG)
                 pDlg->SetCurPageId("wrap");
 
-            if( pDlg->Execute() )
+            if (pDlg->Execute() == RET_OK)
             {
                 rSh.StartAllAction();
                 rSh.StartUndo(UNDO_START);
-- 
2.7.1

